backend/schema/room.py

```diff
@@ -1,15 +1,35 @@
 from datetime import datetime
+from enum import Enum
 from typing import Dict, Optional
 
-from pydantic import BaseModel
+from fastapi import HTTPException
+from pydantic import BaseModel, root_validator, validator
+
+
+class Role(str, Enum):
+    """
+    Enum for the roles of a user in a room.
+    """
+
+    ADMIN = "admin"
+    MEMBER = "member"
+
+
+class Plugin(str, Enum):
+    """
```

Comment:
😁 This docstring too needs a little adjustment
```suggestion
    """Provides list of choices for plugin name."""
    
```

Reply:
Done